drivers/usbdev_synopsys_dwc2/usbdev_synopsys_dwc2.c

```diff
@@ -587,21 +648,32 @@ static void _flush_rx_fifo(const dwc2_usb_otg_fshs_config_t *conf)
 static void _sleep_periph(const dwc2_usb_otg_fshs_config_t *conf)
 {
     *_pcgcctl_reg(conf) |= USB_OTG_PCGCCTL_STOPCLK;
-    /* Unblocking STM32_PM_STOP during suspend on the stm32f446 breaks
-     * while (un)blocking works on the stm32f401, needs more
-     * investigation with a larger set of chips */
-#if defined(STM32_USB_OTG_CID_1x)
+#if defined(MCU_STM32)
```

Comment:
@bergzand I'm not sure whether this change could cause not to work on stm32f446 any longer. I have tested it with a number of non CID 1.x core versions, but couldn't find any problem.